client/app/components/group/_groupMap/groupMap.controller.js

```diff
@@ -0,0 +1,127 @@
+class GroupMapController {
+  constructor($scope, CurrentGroup, CurrentStores, CurrentUsers, $timeout) {
+    "ngInject";
+    Object.assign(this, {
+      $scope,
+      CurrentGroup,
+      CurrentStores,
+      CurrentUsers,
+      $timeout,
+      markers: {},
+      bounds: {},
+      center: {},
+      defaults: {
+        scrollWheelZoom: false
+      },
+      options: {
+        showUsers: false
+      }
+    });
+  }
+
+  $onInit() {
+    // deep watching
+    this.destroy = this.$scope.$watch(() => this.CurrentStores.list, () => {
+      this.$timeout(() => this.update(), 200);
```

Comment:
I'm really not too happy with having these timeouts - but sadly the icons didn't always show up otherwise. Not quite sure what to do...